Consumer Expenditure Survey/download all microdata.R

```diff
@@ -48,6 +48,7 @@ source_url( "https://raw.githubusercontent.com/ajdamico/asdfree/master/Poverty/s
 names( all_thresholds ) <- gsub( "year" , "this_year" , names( all_thresholds ) )
 
 # figure out which years are available to download
+Sys.setlocale('LC_ALL','C') 
```

Comment:
@rodonn sorry i didn't look closer..  i try to avoid setting global settings in the scripts.  would need to be tested on mac/windows/unix, sorry for wasting your time